lib/fibonacci.rb

```diff
@@ -0,0 +1,10 @@
+# Computes the nth fibonacci number in the series starting with 0.
+# fibonacci series: 0 1 1 2 3 5 8 13 21 ...
+# e.g. 0th fibonacci number is 0
+# e.g. 1st fibonacci number is 1
+# ....
+# e.g. 6th fibonacci number is 8
+def fibonacci(n)
+  puts "NOT IMPLEMENTED"
+  return n
```

Comment:
How would you feel about doing `raise NotImplementedError` here instead?

Best practice I think is generally to avoid `puts` statements and other things that will print to the console for code being tested, because that output kind of gets randomly interspersed with the test runner's output.

Reply:
Done!